Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml

```diff
@@ -1177,6 +1223,516 @@ script:
+      - name: serial_number
+        description: The unique serial number assigned to an email message. Use `cisco-esa-message-search` to get the message serial numer.
+        required: true
+      - name: start_date
+        description: Start date for the messages. Timestamp in ISO format or <number> <time unit>, e.g., 2022-01-01T00:00:00.000Z, 12 hours, 7 days, 3 months, now.
+        defaultValue: 1 month
+      - name: end_date
+        description: End date for the messages. Timestamp in ISO format or <number> <time unit>, e.g., 2022-01-01T00:00:00.000Z, 12 hours, 7 days, 3 months, now.
+        defaultValue: now
+      - name: limit
+        description: The maximum number of records to return.
+        defaultValue: "50"
+      - name: all_results
+        description: Whether to retrieve all the results by overriding the default limit.
+        defaultValue: "false"
+    outputs:
+      - contextPath: CiscoESA.MessageConnection.sender_group
+        description: The sender group assigned to the connection (e.g., RELAYLIST).
+        type: String
+      - contextPath: CiscoESA.MessageConnection.Message.timestamp
+        description: Timestamp of the connection summary event.
+        type: String
+      - contextPath: CiscoESA.MessageConnection.Message.description
+        description: Description of the connection summary event.
+        type: String
+      - contextPath: CiscoESA.MessageConnection.Message.last_event
+        description: Indicates whether this is the final summary event.
+        type: Boolean
+      - contextPath: CiscoESA.MessageConnection.sbrs
+        description: Sender Base Reputation Score (SBRS) of the sender IP.
+        type: String
+  - name: cisco-esa-message-remediation-details-get
+    description: Retrieve historical remediation actions taken on messages, such as message deletions or recalls. Includes batch initiator details, message status, delivery timestamps, and whether the message was read, supporting audit and compliance tracking for retroactive threat actions.
+    arguments:
+      - name: injection_connection_id
+        description: An Injection Connection ID (ICID). A numerical identifier for an individual SMTP connection to the system. Use `cisco-esa-message-search` to get message Injection Connection IDs. Consider to insert this parameter in case you don't achive any results.
+        required: false
+      - name: message_ids
+        description: Comma-separated list of message IDs. Use `cisco-esa-message-search` to get message IDs.
+        required: true
+        isArray: true
+        default: true
+      - name: serial_number
+        description: The unique serial number assigned to an email message. Use `cisco-esa-message-search` to get the message serial numer.
+        required: true
+      - name: start_date
+        description: Start date for the messages. Timestamp in ISO format or <number> <time unit>, e.g., 2022-01-01T00:00:00.000Z, 12 hours, 7 days, 3 months, now.
```

Comment:
The start date for the messages. Timestamp in ISO format or <number> <time unit>, for example, 2022-01-01T00:00:00.000Z, 12 hours, 7 days, 3 months, now.